Package.swift

```diff
@@ -131,5 +131,12 @@ let package = Package(
             ],
             path: "Source/SwiftLintCoreMacros"
         ),
+        .testTarget(
+            name: "MacroTests",
```

Comment:
Should we go with `SwiftLintCoreMacrosTests` as the module name?

Reply:
Our other test targets are similarly terse, which I think is fine given that they’re already scoped to this project.